composer.json

```diff
@@ -37,7 +37,12 @@
     "symfony/phpunit-bridge": "~2.7|~3.0|~4.0",
     "symfony/twig-bundle":    "~2.7|~3.0|~4.0",
     "symfony/var-dumper":     "~2.7|~3.0|~4.0",
-    "symfony/yaml":           "~2.7|~3.0|~4.0"
+    "symfony/yaml":           "~2.7|~3.0|~4.0",
+    "namshi/cuzzle":          "^2.0",
+    "phpunit/phpunit":        "^6.5"
```

Comment:
Do we need `"phpunit/phpunit"` here? It's downloaded and managed through `phpunit-bridge`.